test/profiles.http

```diff
@@ -0,0 +1,25 @@
+GET http://localhost3000/profiles
+
+###
```

Comment:
this deserves a true comment (not just "###") to know what this request does :)

Reply:
Fixed, now has a comment.